src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="main__calendar calendar calendar--start-day--sun calendar--month-length--29">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
use nth-child instead extra class
```suggestion
        <div class="calendar__day"></div>
```